src/PrestaShopBundle/Command/ShopCreatorCommand.php

```diff
@@ -0,0 +1,244 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Command;
+
+use Address;
+use Cart;
+use CartRule;
+use Customer;
+use Db;
+use Faker\Factory;
+use Order;
+use OrderState;
+use Product;
+use Shop;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * This command is used for appending the hook names in the configuration file.
+ */
+class ShopCreatorCommand extends Command
+{
+    /**
+     * @var string
+     */
+    private $dbPrefix;
+
+    public function __construct(string $dbPrefix)
+    {
+        parent::__construct();
+        $this->dbPrefix = $dbPrefix;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('prestashop:shop-creator')
+            ->addOption('orders', null, InputOption::VALUE_OPTIONAL, 'Number of orders to create', 10000)
+            ->addOption('customers', null, InputOption::VALUE_OPTIONAL, 'Number of customers without order to create', 5000)
+            ->addOption('carts', null, InputOption::VALUE_OPTIONAL, 'Number of carts to create', 50000)
+            ->addOption('cart-rules', null, InputOption::VALUE_OPTIONAL, 'Number of cart rules to create', 10000)
+            ->addOption('products', null, InputOption::VALUE_OPTIONAL, 'Number of products to create', 10000)
```

Comment:
Good question.
What do you think if we generate by default only 10 orders, 5 customers, 10 carts, 10 cart rules and 20 products?
I think it can be enough for testing, but if the user need more he can use his own values via options.